<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bienvenido al Banco del Pueblo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cuentas">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cuentas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clientes">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modificar_datos">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar Datos Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deposito_retiro">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Depositar/Retirar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crear_cliente">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crear_cuenta">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Cuenta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bloquear">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Bloquear/Desbloquear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="consultar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
